locale/en_US/default.po

```diff
@@ -136,3 +136,11 @@ msgstr "Image"
 
 msgid "default.genres.styleSheet"
 msgstr "HTML Stylesheet"
+
+msgid "default.contextSettings.emailSignature"
+msgstr ""
+"<br/>"
+"<hr /><br/>"
+"<p><a href=\"{$contextUrl}\">{$contextName}</a></p>"
+"<p>{$mailingAddress}</p>"
+"<p>{$contactName} <a href='mailto: {$contactEmail}'>{$contactEmail}</a></p>"
```

Comment:
@dmitris Can you use `\"` instead of `'` for the `href` attribute? Email clients are _really_ picky about the how they parse HTML (much worse than browsers). I haven't found any specific evidence that this will cause a problem, but all of the email templates that I see use `"` instead of `'`, so just to be safe.

Reply:
-> forward to @defstat, the PR author

Reply:
(Welcome to the team, @dmitris! :upside_down_face:)

Reply:
Fixed